TodoListClient/App.config

```diff
@@ -6,9 +6,8 @@
   <appSettings>
     <add key="ida:Tenant" value="[Enter tenant name, e.g. contoso.onmicrosoft.com]" />
     <add key="ida:ClientId" value="[Enter client ID of the TodoListClient-ManualJwt as obtained from Azure Portal, e.g. 82692da5-a86f-44c9-9d53-2f88d52b478b]" />
-    <add key="todo:TodoListResourceId" value="[Enter App ID URI of TodoListService-ManualJwt, e.g. api://{clientID}]" />
+    <add key="todo:TodoListScope" value="[Enter App ID URI of TodoListService-ManualJwt, e.g. api://{clientID}]/access_as_user" />
```

Comment:
>TodoListScope [](http://example.com/codeflow?start=19&length=13)

this change needs to be reflected in the sample.json and the powershell code and steps.md will need to be re-generated

Reply:
I've reverted the change so no configuration is changed